roles/python/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: venv
+  ansible.builtin.apt:
+    name: python3-venv
+  become: true
+
+- name: add deadsnakes ppa
+  ansible.builtin.apt_repository:
+    # https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa
+    repo: ppa:deadsnakes/ppa
+  become: true
+
+- name: additional python versions
+  ansible.builtin.apt:
+    name:
+      - python{{ item }}
+      - python{{ item }}-venv
+      - python{{ item }}-dev
+    update_cache: true
+  loop:
+    - "3.13" # Latest stable
+    # 3.12 is standard python3 on Ubuntu 24.04
+    - "3.9" # Default on Oracle Linux 9
+    - "3.8" # Latest available in software collection on Oracle Linux 7
+  become: true
```

Comment:
Is it not considered best-practice to install python as a non-root user?

Reply:
Yeah it's generally not advisabe to "mess" with system python as root and def not change the default.

So this is just installing python versions "on the side". So we get `python3.8` etc binaries.